src/api/goods.ts

```diff
@@ -1,11 +1,8 @@
+import { Good } from '../react-app-env';
 // eslint-disable-next-line
 const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
 
 export function getAll(): Promise<Good[]> {
   return fetch(API_URL)
     .then(response => response.json());
 }
-
-export const get5First = () => {};
-
-export const getRedGoods = () => {};
```

Comment:
You shouldn't delete this functions
You should use them in your code
get5First function should return an array with 5 elements
getRedGoods function should return an array with red elements